### PR TITLE
Update UmbracoProject.csproj

### PR DESCRIPTION
**Pull request by cornehoskam**
Changing the comment would make the intent of the comment more clear, as you cannot use InMemoryAuto on compiled Razor views, but you're still allowed to keep it as false if you're using a different ModelsBuilder mode.

### Prerequisites

- [X] I have added steps to test this contribution in the description below

If there's an existing issue for this PR then this fixes

### CODE REVIEW

**Review by nathanwoulfe — APPROVED** [review]
Thanks @cornehoskam, your change reads a lot better. Let's get it merged.

**Comment by mikecp** [comment]
Looks blocked on checking if we can merge...

**Review by mikecp — APPROVED** [review]
Approving to see if somehow this restarts the checking process

**Comment by nathanwoulfe** [comment]
Weird... I let this one sit for a bit this morning hoping it would sort itself. Definitely has not sorted itself...

**Pull request merged**
